assets/Script/main.js

```diff
@@ -0,0 +1,83 @@
+import Book, { storageMyLibrary, remove, reloadPage } from './function';
+
+const btn = document.getElementById('submit_btn');
+
+
+function addBookToLibrary() {
+  let myLibrary = JSON.parse(localStorage.getItem('myLibrary'));
+
+  if (myLibrary === null) {
+    myLibrary = [];
+  }
+
+  const form = document.getElementById('input_book');
+  const bookTitle = form.elements[0].value;
+  const bookAuthor = form.elements[1].value;
+  const bookPages = form.elements[2].value;
+  const bookRead = form.elements[3].value;
+  const book3 = new Book(bookAuthor, bookTitle, bookPages, bookRead);
+
+  myLibrary.push(book3);
+
+  storageMyLibrary(myLibrary);
+  reloadPage();
+}
+btn.addEventListener('click', () => {
+  addBookToLibrary();
```

Comment:
## Fix book creation 
- [ ] As a user, I expect the form to be submitting the form values and saving my book. It is currently throwing an error at the console when a user tries to save a book. below I have shared a screenshot of the same about these specific errors. Take a look and fix this, kindly, thank you.

![library](https://user-images.githubusercontent.com/22345961/111663739-25d65080-8822-11eb-9f55-b2fa42b67dc0.png)